daemon/cmd/daemon_main.go

```diff
@@ -994,8 +994,14 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	)
 	option.BindEnv(vp, option.HubbleMonitorEvents)
 
-	flags.StringSlice(option.HubbleRedact, []string{}, "List of Hubble redact options")
-	option.BindEnv(vp, option.HubbleRedact)
+	flags.Bool(option.HubbleRedactEnabled, defaults.HubbleRedactEnabled, "Hubble redact sensitive information from flows")
```

Comment:
Do we really need a separate CLI option for this? Couldn't we just assume this to be enabled whether either of the following two options (http RUL query and/or kafka API key redaction) is enabled, i.e. replace the single check for `option.Config.HubbleRedactEnabled` by `option.Config.HubbleRedactHttpURLQuery || option.Config.HubbleRedactKafkaApiKey`?

Reply:
That's based on "this approach would provide the best flexibility for the user yet with reasonable default, all the user has to do is to set hubble.redact.enabled=true" from https://github.com/cilium/cilium/issues/23887#issuecomment-1632167762.

Reply:
Makes sense, thanks.